### PR TITLE
Updated pia-back to ruby 2.6.5 and rails 6.0.2 

### PR DESCRIPTION
**Pull request by kosmas58**
- Updated to ruby 2.6.5 and rails 6.0.2
- Replaced FactoryGirl through FactoryBot
- Addded docker configuration for testing

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+REDIS_URL="redis://localhost:6379/1"
+ALLOWED_CORS_ORIGINS='*'
+RAILS_SERVE_STATIC_FILES='true'
```
Comment (brunto) [inline .env]: Do not add the .env file to the repository.

File: .gitignore [hunk 2]
```diff
@@ -21,18 +30,19 @@
 .byebug_history
 
 # Ignore database configuration
-config/database.yml
+# config/database.yml
```
Comment (brunto) [inline .gitignore]: Do not add database.yml to the repository.

Comment (kosmas58) [reply .gitignore]: Has been removed and added to .gitignore

File: .env (hunk 1 above)
Comment (kosmas58) [reply .env]: Has been removed and added to .gitignore

**Pull request merged**